### PR TITLE
Added `transform-origin`

### PR DESCRIPTION
**Pull request by jmswisher**
- New in Fx 77, per https://bugzil.la/1581691
- Description of that bug says both Blink and Webkit support it.

### CODE REVIEW

File: svg/attributes/presentation.json [hunk 1]
```diff
@@ -2882,6 +2882,54 @@
             }
           }
         },
+        "transform-origin": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/SVG/Attribute/transform-origin",
+            "support": {
+              "chrome": {
+                "version_added": "yes"
```
Comment (vinyldarkscratch) [inline svg/attributes/presentation.json:2890]: Looks like you were trying to use `"yes"` instead of `true`.  😛

**Comment by jmswisher** [comment]
Nevermind. I'm not going to chase down versions for other browsers, given that data for SVG are not displayed on MDN.

**Comment by vinyldarkscratch** [comment]
We can chase down the exact versions another time!  Actually what I was trying to say what that if you're not certain what the version is, then we should put `true` as the value.  `"yes"` was the older syntax.

**Comment by younggun0509** [comment]
Thank you for assisting me with this

And what should I do?

Thank you so much
Nizar

**Pull request closed without merging**